src/dev/impl/DevToys/DevToys.csproj

```diff
@@ -468,6 +474,29 @@
     <AppxManifest Include="Package.appxmanifest">
       <SubType>Designer</SubType>
     </AppxManifest>
+    <PRIResource Include="Strings\en-US\XmlFormatter.resw" />
```

Comment:
Can you please remove these PRIResource  lines? Line 500 includes all of them automatically. :)

Reply:
Done.